source/miscellaneous.md

```diff
@@ -0,0 +1,31 @@
+(c:misc)=
+# Miscellaneous
+
+%---------------------------------------------------------------------------------------------------
+(s:set)=
+## Set command
+
+The `set` command is used for setting parameters. The components of `set` are:
+```{code} yaml
+parameter     # [String] Parmeter(s) to vary.
+value         # [Expression] Value to set.
+delta         # [Boolean] Default false. If false, parameter is set to value.
+              #   If true, value is added to existing value of parameter.
+```
+In the `value` expression, the symbol `@` can be used for the value of the parameter being changed.
+Example:
+```{code} yaml
+- Q1:
+    kind: Quadrupole
+    ...
+
+- set:
+    parameter: Q1.*>MagneticMultipoleP.Kn3
+    value: 2 * @ 
+
+- Lattice:
+  - branches:
+      - line1         # First branch
```

Comment:
This seems to be the only functional change of this PR, while the rest is restructuring?

Can you please submit this in a separate PR to keep purely formatting and features separate for reviews? :pray: 
Mixed-purpose PRs like this tend to otherwise cause a lot of confusion and run long to review/merge.

Reply:
Done.